package.json

```diff
@@ -76,32 +76,26 @@
         },
         "coverage-gutters.gutterIconPathLight": {
           "type": "string",
-          "default": "./app_images/gutter-icon-light.svg",
           "description": "path to an icon (svg, png, etc) for displaying in the gutter for full coverage"
         },
         "coverage-gutters.gutterIconPathDark": {
           "type": "string",
-          "default": "./app_images/gutter-icon-dark.svg",
           "description": "path to an icon (svg, png, etc) for displaying in the gutter for full coverage"
         },
         "coverage-gutters.partialGutterIconPathLight": {
           "type": "string",
-          "default": "./app_images/partial-gutter-icon-light.svg",
           "description": "path to an icon (svg, png, etc) for displaying in the gutter for partial coverage"
         },
         "coverage-gutters.partialGutterIconPathDark": {
           "type": "string",
-          "default": "./app_images/partial-gutter-icon-dark.svg",
           "description": "path to an icon (svg, png, etc) for displaying in the gutter for partial coverage"
         },
         "coverage-gutters.noGutterIconPathLight": {
           "type": "string",
-          "default": "./app_images/no-gutter-icon-light.svg",
           "description": "path to an icon (svg, png, etc) for displaying in the gutter for no coverage"
         },
         "coverage-gutters.noGutterIconPathDark": {
           "type": "string",
-          "default": "./app_images/no-gutter-icon-dark.svg",
```

Comment:
I dont believe moving the defaults for these icon paths into the source code will force us to do a breaking version change 🤔.
It will make it appear like you dont get a default in the config settings I think. I don't believe many will notice the change though.
![image](https://user-images.githubusercontent.com/1335972/119281175-8484c580-bbe9-11eb-9439-c647a68309d6.png)

Reply:
I built the `vsix` and installed locally to have a look / double check, when no default is provided this is what is shown on screen (see `coverage-gutters.gutterIconPathLight`:

![image](https://user-images.githubusercontent.com/37993418/119322707-3d6a0380-bcc1-11eb-9d7b-c4fcdcd9ac2c.png)

From the `WorkspaceConfiguration` API:

```
		/**
		 * Return a value from this configuration.
		 *
		 * @param section Configuration name, supports _dotted_ names.
		 * @param defaultValue A value should be returned when no value could be found, is `undefined`.
		 * @return The value `section` denotes or the default.
		 */
		get<T>(section: string, defaultValue: T): T;
```

If no value is found `undefined` will be returned and the code will work as expected so we can get away without a default.

The potential breaking change will be to users who have figured out that they can save their own custom SVGs with a path relative to where the extension is installed. I don't see that as much of an issue as I think only a handful of people would have worked it out.

Potential we could change the wording in the setting to `path to a custom icon (svg, png, etc) for displaying in the gutter...` which would implicitly let people know that there is a default.

LMK what you think.

Thanks,